src/helpers/FilterData.js

```diff
@@ -0,0 +1,9 @@
+export const FilterData = (data, filterState) => {
+  return data
+    .filter((item) => +item.price >= +filterState.price)
```

Comment:
The products to be displayed should be of price lesser than the 'filterState.price' for the slider. Please check.